client/src/components/utils/Footer/Footer.js

```diff
@@ -4,7 +4,7 @@ import { Anchor, Footer as SiteFooter, Text } from "grommet";
 export default function Footer() {
   return (
     <SiteFooter
-      style={{ padding: 20, position: "fixed", bottom: 0, width: "100vw" }}
+      style={{ padding: 20, position: "static", bottom: 0, width: "100vw" }}
```

Comment:
Revert this

Reply:
static is correct, fixed is the foot is always at the bottom of the screen
we don't want that

Reply:
It doesn't look good on my screen